File: lib/java/windows/src/main/java/alphaTab/alphaSkia/AlphaSkiaWindows.java
package alphaTab.alphaSkia;

/**
 * The alphaSkia initializer for Windows.
 */
public class AlphaSkiaWindows extends AlphaSkiaJre {
    /**
     * The alphaSkia initializer for Windows.
     */
    public static final AlphaSkiaWindows INSTANCE = new AlphaSkiaWindows();

    private AlphaSkiaWindows() {
    }

    @Override
    protected String[] getJavaResources() {
        return new String[]{
                "/native/win-" + getCurrentArchitecture() + "/libalphaskiajni.dll"
        };
    }
}

File: lib/kotlin/windows/src/main/java/alphaTab/alphaSkia/AlphaSkiaWindows.kt
package alphaTab.alphaSkia

/**
 * The alphaSkia initializer for Windows.
 */
class AlphaSkiaWindows private constructor() : AlphaSkiaJre() {
    override fun getJavaResources(): Array<String> {
        return arrayOf(
            "/native/win-" + getCurrentArchitecture() + "/libalphaskiajni.dll"
        )
    }

    companion object {
        /**
         * The alphaSkia initializer for Windows.
         */
        @JvmField
        val INSTANCE = AlphaSkiaWindows()
    }
}
